### ISSUE TITLE
Text encoding failure

### ISSUE THREAD

**Issue by AntoineTurmel** [classam/pierc#36]
When someone use XChat to publish with  ISO8859-1(5) and type "ç" it stores "ç" on the table main column "message" causing "Something Has Gone Spectacularly Wrong!"
if the charset is set to UTF-8, it stores "Ã§" for "ç" and do not cause error...

Is it possible to handle both ?

**Comment by frdmn** [comment]
If I understand you right, only the front-end renders said character wrong? But the database contains the proper  "ç"?

**Comment by AntoineTurmel** [comment]
yeah the front-end shows "Something Has Gone Spectacularly Wrong!" when it's a "ç" stored
When "Ã§" is stored on the database, the front-end shows "ç"...

**Comment by frdmn** [comment]
What would happen if you try to convert to UTF-8 [before writing into the database](https://github.com/classam/pierc/blob/master/bot/pierc_db.py#L43-L49):

```python
    query = "INSERT INTO main (channel, name, time, message, type, hidden) VALUES" + \
    "(\""+unicode(self.conn.escape_string(channel), 'utf-8')+ "\"," + \
    "\""+unicode(self.conn.escape_string(name), 'utf-8')+"\"," + \
    "\""+time+"\"," + \
    "\""+unicode(self.conn.escape_string(message), 'utf-8')+"\"," + \
    "\""+self.conn.escape_string(msgtype)+"\"," + \
    "\""+self.conn.escape_string(hidden)+"\")"
```

**Comment by AntoineTurmel** [comment]
I'll try thanks !

**Comment by AntoineTurmel** [comment]
Ok now my text is stored well, but I still have the error but resolved adding this in pierc_db.php:
```mysqli_set_charset($this->_conn, "utf8");```

**Comment by frdmn** [comment]
Great. You can create a PR if you like. I think others might profit from those changes as well :)
